tap_hubspot_beta/client_base.py

```diff
@@ -178,7 +249,9 @@ def validate_response(self, response: requests.Response) -> None:
                 curl_command = curlify.to_curl(response.request)
                 logging.error(f"Response code: {response.status_code}, info: {response.text}")
                 logging.error(f"CURL command for failed request: {curl_command}")
-            raise RetriableAPIError(msg)
+                logging.error("Sleeping for 60 seconds before retrying...")
+                time.sleep(60)
```

Comment:
why put this time here instead of change the backoff?

Reply:
I just wanted to add the extra time for when a 500 happened and not change the whole backoff behavior. What would you suggest?

Reply:
Looking at the failing job, from the first to the last failure, we don't have more than 1 minute, where the last chain of failures occasioned the job to fail.